node-module-system/a-wrapped-module.js

```diff
@@ -0,0 +1,10 @@
+console.log('exports typeof: ', typeof exports);
```

Comment:
console.log inserts a space between arguments, so the space at the end of the message string isn't needed. (Trivial, though!)